### PR TITLE
[Release] last internal review before pull request to babelsberg/babelsberg-js

### PR DESCRIPTION
**Pull request by j4yk**
Final call for review and corrections, otherwise will open PR against babelsberg/babelsberg-js on Monday or Tuesday. :+1: if approved.

### CODE REVIEW

**Comment by j4yk** [comment]
note to self: always push before opening PRs on Github...

**Comment by gersseba** [comment]
Should we push our readme changes?

File: babelsberg/cassowary_ext.js [hunk 1]
```diff
@@ -26,6 +26,15 @@ ClSimplexSolver.addMethods({
         var constraint = new Constraint(func, this);
         constraint.priority = priority;
         return constraint;
+    },
+    solverName: 'Cassowary',
+    supportsMethods: function() { return false; },
+    supportsSoftConstraints: function() { return true; },
+    supportsFiniteDomains: function() { return false; },
+    supportedDataTypes: function() {
+        // Cassowary does not support strings, but there are actively used scenarios
```
Comment (gersseba) [inline babelsberg/cassowary_ext.js]: but THEY are actively

Comment (j4yk) [inline babelsberg/cassowary_ext.js:35]: "but there are actively used scenarios"... seems about right, we might reword it to "but there are scenarios where the coercion [...] is deliberately used"

Comment (gersseba) [inline babelsberg/cassowary_ext.js:35]: nevermind, didn't read the whole sentence...

**Comment by j4yk** [comment]
Posted pull request to upstream two days ago: https://github.com/babelsberg/babelsberg-js/pull/26
